src/contexts/PeopleContexts.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {
+  createContext,
+  useState,
+  useCallback,
+  useMemo,
+  ReactNode,
+} from 'react';
+import debounce from 'lodash/debounce';
```

Comment:
Try to fix eslint error. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review
![image](https://github.com/mate-academy/react_autocomplete/assets/70009205/41b171fc-b603-409b-a16a-f5b4ef2068c0)